influunt-app/app/scripts/services/util/dropzone.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+/**
+ * @ngdoc service
+ * @name influuntApp.dropzoneUtils
+ * @description
+ * # dropzoneUtils
+ * Factory in the influuntApp.
+ */
+angular.module('influuntApp')
+  .factory('dropzoneUtils', function () {
+
+    var countFiles = function(element, maxFiles) {
+      maxFiles = maxFiles || Infinity;
+      var thisDropzone = $(element);
+      var files = thisDropzone.find('.dz-preview');
+      return files.length >= maxFiles ? thisDropzone.find('.dropzone-area').hide() : thisDropzone.find('.dropzone-area').show();
```

Comment:
Somente a área de upload de arquivos deverá ser escondida. Toda a estrutura restante deverá permanecer na tela. Na prática, isto deverá somente impedir o usuário de tentar fazer mais um upload quando o limite for atingido.